### DISCUSSION TITLE
Password Expiration and Limit Reuse of existing passwords.

### DISCUSSION THREAD

**Discussion by xorkh** [directus/directus#16012] [Feature Requests]
I'm an architect looking for a headless CMS for our next platform. Directus checks a lot of boxes but a mandatory requirement for this new platform is additional security controls such as: 

1. When logging into the CMS or API end points check if the password is expired. Require change of password before allowing any operations. 
2. When changing or resetting password user cannot use the last X passwords.

Can this be done natively within Directus? If not can an extension be implemented to handle such requirements.

**Comment by rijkvanzanten** [comment]
This is not natively available, but should be pretty easy to spin up through extensions or even flows 👍🏻

**Reply by xorkh** [reply]
Thanks for the quick response. I'll try that for sure. :)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
